lib/liftoff/scheme_builder.rb

```diff
@@ -0,0 +1,50 @@
+module Liftoff
+  class SchemeBuilder
+
+    def initialize(xcode_project, config)
+      @xcode_project = xcode_project
+      @config = config
+    end
+
+    def create_schemes
+      @xcode_project.generate_scheme # generates the default scheme
```

Comment:
What do you think about renaming the method itself to `generate_default_scheme` and removing the comment?